client/components/blogging-prompt/prompts-navigation.jsx

```diff
@@ -61,6 +59,7 @@ const PromptsNavigation = ( { prompts } ) => {
 			recordTracksEvent( `calypso_customer_home_answer_prompt`, {
```

Comment:
I just noticed that the event names here use "customer_home" 🙁. That's very deceptive to someone browsing events since these clicks may come from elsewhere... I think we need to address it sooner rather than later by introducing new event names that don't assume the context of the prompts.

Reply:
Yeah good catch. I'm going to leave that to a follow up PR though.

Reply:
I added a function to prefix the tracks events based on the `viewContext`

Reply:
I will need to register the new events when they show up.